chocolatey/plugins/modules/win_chocolatey_feature.ps1

```diff
@@ -39,26 +46,46 @@ $spec = Get-ModuleSpec
 $module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
 Set-ActiveModule $module
 
-$name = $module.Params.name
-$state = $module.Params.state
+$featuresToSet = if ($module.Params.features) {
+    $module.Params.features
+} else {
```

Comment:
We are using cuddled else's here. Nowhere else in the code (that I could see) is using that. We need to make sure we are consistent in our code.